app/views/layouts/_cookie_banner.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="cookieBanner" class="hidden">
+    <p>We use some essential cookies to make this service work.</p>
+    <p>We'd also like to use analytics cookies so we can understand how you use the service and make improvements.</p>    <a href="/cookies" id="cookies-eu-more">Read more</a>
+    <button id="rejectCookies">Reject</button>
```

Comment:
What's the difference between "Reject" and "Reject analytics cookies" in practical terms? Are we able to not set `session[:uuid]` and `session[:just_rated_place_id]`, and do want to offer this as an option?

And what about the cookie that remembers they rejected cookies? (😬 )

Reply:
Whoops - that first bit was a rebase gone bad. I'll need to add the `just_rated_place_id` to the strictly necessary cookies on the cookie page.